### PR TITLE
Fix test syntax

### PR DESCRIPTION
**Pull request by baude**
Remove the use of bash -c unless it involves a bash pipe. Also, use
run test_runner.sh inside script so that centos can be happy.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM! Let's hope this helps end our testing woes

File: test/podman_commit.bats [hunk 1]
```diff
@@ -13,109 +13,85 @@ function setup() {
...
     echo "$output"
     [ "$status" -eq 0 ]
     run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} images | grep image-committed"
     echo "$output"
     [ "$status" -eq 0 ]
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} rmi image-committed"
-    echo "$output"
-    [ "$status" -eq 0 ]
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} rm my_ctr"
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} rmi image-committed
```
Comment (TomSweeneyRedHat) [inline test/podman_commit.bats]: Just curious why you removed the tests for the stops, runs, creates from this module but not the others?   Personally I would have kept them here, but shrug.

**Comment by TomSweeneyRedHat** [comment]
Red Hat CI is not happy in a few tests.

File: test/podman_commit.bats (hunk 1 above)
Comment (baude) [reply test/podman_commit.bats]: nothing particular ... i didnt remove any tests.  in cases where the commands results are not explicitly tied to what is being tested, it doesnt need to be preceeded with a run command,  nor its output or status checked.  for example, if a test of mine runs a container first thing so I can stop it, technically i dont need to preceed that container run with 'run'.

**Comment by baude** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**